tests/app/service/test_rest.py

```diff
@@ -216,6 +216,10 @@ def test_create_service(client, sample_user):
     assert json_resp['data']['dvla_organisation'] == '001'
     assert json_resp['data']['sms_sender'] == current_app.config['FROM_NUMBER']
 
+    service_db = Service.query.get(json_resp['data']['id'])
+    assert service_db.name == 'created service'
```

Comment:
Can we have a test clause somewhere for that marshmallow fakery in the to_json method on the model object? Or is that covered by other tests?